### DISCUSSION TITLE
Getting this tool more to the public eye

### DISCUSSION THREAD

**Discussion by Landy005** [Mojang/minecraft-editor#200] [Feedback]
I have the feeling when I talk about the editor to other bedrock builders or creators they don't know what it is.

I run a bedrock building community by providing a public building community with geyser so bedrock builders can join and create using the java tools from their bedrock platform. From what I see and can tell bedrock users download worldedit "mods" for their phones or computer to try to imitate the tools java have.
I don't want to stamp this as the 'axiom' imitation but so far the whole building community is aware and familiar with axiom. Even though axiom came out after the editor, the axiom discord has 50 thousand members just for perspective (You'll probably be stamped as 'Mojang trying to bring axiom to bedrock' since they got in the public eye first), and even though this tool is public I think it could use the marketing they had which will help you get way more feedback from actual users and builders. Because those are the ones that will use every single function of this tool everyday. 
They're just pretty unaware at this moment, or they know but found out they can't have it on their phones. If you have windows 10 you most likely have java and will use the java tools to create. 

So if you want more feedback or spread this tool to the target audience I think there are a few steps that can be taken

Study the way Moulberry advertised axiom. 
- He reached out to relevant youtubers to cover this, making connections in the java building community by collaborating with the 2 biggest creative building servers 'the bakery' and 'builders refuge' to use try and the tool for free. 

- They also have a discord to interact with their playerbase. Github's target might not reach those who you'd actually want to reach. It's convenient for the mojang team but perhaps less convenient for the actual builders. I've forwarded many people here to make feature requests but they never got around to create their github accounts. Discord has a low entry form of communication and doesn't need alot of eccentric motivation to connect from their perspective, this way more. 

- Moulberry also makes videos of new features and posts them on social media, while I have not seen one editor video being posted. Which is why I'm wondering; why isn't this being properly marketed yet? 

- And finally, when builders create something amazing with this tool, the tool itself gets highlighted in return! The best form of marketing is simply word of mouth, which I'm guessing mojang is betting on. If this ever gets implemented in the main bedrock versions I'm sure it will but as long as it's in preview I'm not too sure this is reaching the audience this is made for.

**Comment by Landy005** [comment]
Would be awesome to have tutorials for various things on
https://www.youtube.com/@MinecraftCreatorChannel
:D

**Comment by bryhaw** [comment]
Hey Landy005, these are great suggestions! We do have a few things in the works that I hope will help with a lot of what you are saying. 

We do have [#editor-mode](https://discordapp.com/channels/523663022053392405/1084090299120373760) in the Bedrock Add-Ons Discord Server. However, this likely is not the right long-term solution. I do think there are tremendous opportunities to build a community around Editor that I would love to see in the not-too-distant future.

**Reply by Landy005** [reply]
Awesome can't wait for what's coming! I think opening an addon discord would be really good and I'd help promote it 100%. 

I might be a little biased as a builder but I'm not too sure if the addon discord is diversified enough to make use of some of the tools. By that I mean that I feel that although this is for everyone, the ones that will make most use of it are the bedrock builders as this is also (mainly but not exclusively) a world building tool.

Java tool communities have always been a hub to connect to other builders. Personally what got me connected with other builders was finding tutorials and communities using tools to build via youtube or social media which nowadays would be for example discord. But that need for connection made me the creator and eventually partner I am today. 
Alot of other creators (and partners) I know started out as builders using these tools and I've yet to meet a developer for java that knows the intricate ways of an angular based tree array placement with world edit commands. And again, as surely there are many other creators using these tools, the real experts on creator plugins like fawe, voxelsniper, arceon, metaedits, schematicbrushreborn, buildertools, gobrush, gopaint, axiom and the recent ezedits are all builders. Alot of these developers are also builders or heavily tied to builders or buildteams. Not so much the other niches. So just continuing that line of thought slightly makes me edge with suggesting trying out promoting this more to builders to see how they would use it and what would create with it. 😁😁😁
